birthday/birthday.py

```diff
@@ -0,0 +1,284 @@
+import logging
+import hashlib
+import asyncio
+import contextlib
+from dateutil import parser
+import datetime
+import discord
+import itertools
+from typing import (
+    Any,
+    Dict,
+)
+
+
+from redbot.core import commands, Config, checks
+from redbot.core.bot import Red
+from redbot.core.config import _ValueCtxManager, Group, Value
+from redbot.core.i18n import Translator, cog_i18n
+from redbot.core.commands import Context, Cog
+
+T_ = Translator("Birthdays", __file__)
+
+
+def _(s):
+    def func(*args, **kwargs):
+        real_args = list(args)
+        real_args.pop(0)
+        return T_(s).format(*real_args, **kwargs)
+
+    return func
+
+
+@cog_i18n(T_)
+class Birthdays(Cog):
+    """Announces people's birthdays and gives them a birthday role for the whole day"""
+
+    __author__ = "PancakeSparkle#8243"
+
+    # Just some constants
+    DATE_GROUP = "DATE"
+    GUILD_DATE_GROUP = "GUILD_DATE"
+
+    # More constants
+    BDAY_LIST_TITLE = _("Birthday List")
+
+    # Even more constants
+    BDAY_WITH_YEAR = _("<@!{}> is now **{} years old**. <:aureliahappy:548738609763713035>")
```

Comment:
you don't need to format this with an id, members have a mention attribute to get the mention string for the member. so it should be .format(member.mention) Same for the other lines